packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json

```diff
@@ -7,8 +7,9 @@
   "pci_databases_general_information_add_nodes": "Ajouter un nœud",
   "pci_databases_general_information_delete_nodes": "Supprimer un nœud",
   "pci_databases_general_information_node_type_label": "Modèle de nœud",
-  "pci_databases_general_information_node_type": "{{coreNumber}} CPU / {{memorySize}} RAM / {{storageSize}} Stockage SSD",
-  "pci_databases_general_information_node_type_disk_range": "{{coreNumber}} CPU / {{memorySize}} RAM / de {{storageMinSize}} à {{storageMaxSize}} Stockage SSD",
+  "pci_databases_general_information_node_type_cpu_memory": "{{coreNumber}} CPU / {{memorySize}} RAM",
+  "pci_databases_general_information_node_type_storage": " / {{storageSize}} de stockage SSD",
```

Comment:
**issues(blocking):** Could you please avoid using `/` (separator) inside translation file?

Doesn't make sense to start a translation by `<SPACE>/<SPACE>{{dynamicValue}}<TEXT_TO_TRANSLATE>`.

Reply:
Hi @antleblanc, we already had the separator in the previous translation. Here we want the possibility to hide the storage, so I simply cut the translation in two, and we display it like this:

- storage > 0: `{{coreNumber}} CPU / {{memorySize}} RAM / {{storageSize}} de stockage SSD` (so both translations side to side)
- storage = 0: `{{coreNumber}} CPU / {{memorySize}} RAM` (so only the first half

I cannot place the separator in the first half, or we will display `{{coreNumber}} CPU / {{memorySize}} RAM / `. 
What do you think would be a better solution ?
Thanks !

Reply:
Separator shouldn't be used this way with translation files.

Here is my recommandation (example):

```html
<span data-translate="key_1"><span>
<span
  data-ng-if="storage > 0"
  data-ng-bind="' / ' + ( 'key_2' | translate:{ coreNumber: 'value', ... } )"></span>
</span>
```

This allow you to store into your translation file following content:

```json
{
  "key_2": "TEXT_HERE"
}
```

Instead of:

```json
{
  "key_2": " / TEXT_HERE"
}
```

Reply:
Thank you @antleblanc , I've change the code using your recommandation: https://github.com/ovh/manager/pull/5984/commits/27a6269da94f31d88605a6adb87cf24ab9f974d4